jsonnet/kube-thanos/thanos.libsonnet

```diff
@@ -0,0 +1,8 @@
+{
+  bucket: (import 'kube-thanos-bucket.libsonnet'),
+  compactor: (import 'kube-thanos-compactor.libsonnet'),
+  querier: (import 'kube-thanos-querier.libsonnet'),
+  receive: (import 'kube-thanos-receive.libsonnet'),
```

Comment:
This should be `receiver`.

Reply:
I think I might have misunderstood the outcome of the issue. I don’t feel this is right.